### PR TITLE
Fix scan-asset relationship and query asset

### PR DESCRIPTION
**Pull request by elyousfi5**

### CODE REVIEW

File: src/ostorlab/serve_app/types.py [hunk 1]
```diff
@@ -315,7 +315,7 @@ class OxoScanType(graphene_sqlalchemy.SQLAlchemyObjectType):
     )
     message_status = graphene.String()
     progress = graphene.String()
-    asset_instance = graphene.Field(OxoAssetType)
+    asset = graphene.String()
```
Comment (3asm) [inline src/ostorlab/serve_app/types.py:318]: !!!

**Review by 3asm — COMMENTED** [review]
I don't understand this PR.

File: src/ostorlab/serve_app/types.py [hunk 2]
```diff
@@ -207,6 +208,14 @@ class Meta:
...
         fields = ("id", "package_name", "application_name")
 
+    def resolve_package_name(self, info: graphql_base.ResolveInfo):
+        with models.Database() as session:
+            return session.query(models.AndroidStore).get(self.id).package_name
+
+    def resolve_application_name(self, info: graphql_base.ResolveInfo):
+        with models.Database() as session:
+            return session.query(models.AndroidStore).get(self.id).application_name
+
```
Comment (mohsinenar) [inline src/ostorlab/serve_app/types.py:218]: Types please!

File: src/ostorlab/serve_app/types.py [hunk 3]
```diff
@@ -218,6 +227,14 @@ class Meta:
         model = models.IosStore
         fields = ("id", "bundle_id", "application_name")
 
+    def resolve_bundle_id(self, info: graphql_base.ResolveInfo):
+        with models.Database() as session:
+            return session.query(models.IosStore).get(self.id).bundle_id
+
+    def resolve_application_name(self, info: graphql_base.ResolveInfo):
+        with models.Database() as session:
+            return session.query(models.IosStore).get(self.id).application_name
```
Comment (mohsinenar) [inline src/ostorlab/serve_app/types.py:236]: Types please!

File: src/ostorlab/serve_app/types.py [hunk 4]
```diff
@@ -256,10 +289,12 @@ class Meta:
         fields = ("id",)
 
     def resolve_links(self, info):
```
Comment (mohsinenar) [inline src/ostorlab/serve_app/types.py:291]: Update the return type please!

**Review by mohsinenar — CHANGES_REQUESTED** [review]

**Review by 3asm — DISMISSED** [review]

**Review by 3asm — APPROVED** [review]

**Review by mohsinenar — APPROVED** [review]

**Pull request merged**